packages/desktopjs-electron/tsconfig-staging.json

```diff
@@ -1,20 +1,24 @@
 {
     "compilerOptions": {
-        "target": "es5",
+        "target": "es6",
```

Comment:
This is staging project used for tests and dts generation, not the actual build which is still es5/rollup -> umd.

Reply:
Changed back to es5